### ISSUE TITLE
analytics.js not being reported as blocked via shields on some websites (polyfilled vs blocked)

### ISSUE THREAD

**Issue by anotherzen** [brave/brave-browser#5693]
## Description 
Not sure if this is by design or not.

I am seeing on some sites that are using analytics.js, the shield is not reporting them blocked or they might not even be blocked. Specifically i was looking at https://bitwarden.com/ where on the previous version of Brave it would show a 1 on the Shield. But on v0.67.125 there is nothing being shown as blocked. This happens on some other sites as well but BitWarden is a good example as it only has 1 item that should be blocked as far as i know.

I am unable to rollback to an older version to show screenshots as i can't find an older installer.

I however do not see the typical: collect?v= listing in the network inspector tab in Brave, like i do if i check with Edge. I do not know if that means anything or not.

## Steps to Reproduce
   1. Go to https://bitwarden.com/
   2. Check and see if the Shield is blocking analytics.js or not.

## Actual result:
Nothing is being reported blocked.

## Expected result:
analytics.js should be reported blocked, as it was shown as blocked on earlier versions of Brave.

## Reproduces how often: 
Easily reproduced

## Brave version (brave://version info)
v0.67.125

## Version/Channel Information:
- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the dev channel? Not tested
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:
I confirmed by a 3rd party that the Shield produced a 1 and analytics.js was the thing being blocked on an earlier version of Brave (not 100% sure what version it was, but most like the one or the other right before v0.67.125)

## Miscellaneous Information:
If you RightClick on the BitWarden website and click Inspect, then go to Network on the top tabs, click Other on the menu row below and then do a CTRL+R you will see that analytics.js is catched. There is however no collect?v= listing.

**Comment by rebron** [comment]
cc: @ryanbr @snyderp

**Comment by snyderp** [comment]
@rebron this is because we're polyfilling analytics, so shields doesn't see it as blocked.  Shields is successfully preventing Google's GA code from loading though.

Probably would be best to both polyfill and also report as blocked in shields.  Prob best to open in a new issue and pull in a shields engineer?

**Comment by ryanbr** [comment]
@anotherzen this should be resolved, showing following blocked items. 

```
https://www.googletagmanager.com/gtm.js?id=GTM-NF32QCS
https://cmp.osano.com/16BZFqRx1MZfx2O4V/779c3da6-bffb-49bb-99b5-7e07b4c65269/osano.js
https://www.google-analytics.com/analytics.js
```

Safe to close ticket @anotherzen  ?

**Comment by anotherzen** [comment]
It's safe to close. I am closing it now.
